articles/macos-setup-experience.md

```diff
@@ -205,6 +205,20 @@ Fleet also provides a REST API for managing setup experience software and script
 
 To manage setup experience software and script using Fleet's best practice GitOps, check out the `macos_setup` key in the GitOps reference documentation [here](https://fleetdm.com/docs/configuration/yaml-files#macos-setup)
 
+## Advanced
+
+> **Experimental feature**. This feature is undergoing rapid improvement, which may result in breaking changes to the API or configuration surface. It is not recommended for use in automated workflows.
+
+By default, Fleet installs the fleetd agent during automatic enrollment (ADE) on macOS hosts. If you need to customize the fleetd configuration, follow these steps to install it manually using a bootstrap package.
+
+First, generate your fleetd package by running the command `fleetctl package`, ensuring you do **not** use the `--use-system-configuration` flag. 
+
+Next, incorporate the generated fleetd package into your bootstrap package and add a script to modify fleetd's [launchd template](https://github.com/fleetdm/fleet/blob/fleet-v4.66.0/orbit/pkg/packaging/macos_templates.go#L96) with the required options.
+
+Then, navigate to **Controls > Setup Experience > Bootstrap package** in the Fleet interface and upload your bootstrap package. Make sure to check the option **Install Fleet’s agent (fleetd) manually** and then select **Save**.
+
+Once the option to manually install Fleet's agent is checked, instead of using **Install software** and **Run script** options, include your software in the bootstrap package.
```

Comment:
@marko-lisica I think let's pull this up higher in the section. That way, the user knows, before they add the bootstrap package, if they do down the manual fleetd route, they have to do software and scripts in their bootstrap package too.